src/TestSuite/IntegrationTestCase.php

```diff
@@ -474,6 +474,25 @@ public function assertRedirect($url = null, $message = '')
     }
 
     /**
+     * Asserts that the Location header contains a substring
+     *
+     * @param string $url The URL you expected the client to go to.
+     * @param string $message The failure message that will be appended to the generated message.
+     * @return void
+     */
+    public function assertRedirectContains($url = null, $message = '')
```

Comment:
We should not use '= null' default for URL. The first argument cannot be empty for the method to make sense.